mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
+	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
+	METHOD add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 value
+	METHOD method_10533 addElement (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10535 setElement (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10601 getHeldType ()B
```

Comment:
```suggestion
	METHOD method_10601 getElementType ()B
```
The name `getElementType` reads better than `getElementId`. The javadoc describes the specific return type.

Reply:
Element type is indistinguishable from the actual element type, which is `9` for lists. There should be a clear difference between the list itself and its contents.

Reply:
Given two methods `NbtList#getType` and `NbtList#getElementType` I feel it is clear which is which (especially given `getType` would return `NbtType` and `getElementType` would return `byte`)